packages/front-end/components/Features/ScheduleInputs.tsx

```diff
@@ -86,93 +87,137 @@ export default function ScheduleInputs(props: Props) {
           Automatically enable and disable an override rule.
         </span>
         <ul className={styles.conditionslist}>
-          {rules.map(({ timestamp, enableFeature }, i) => {
-            const onChange = (value, property, i) => {
-              const newRules = [...rules];
-              newRules[i][property] = value;
-              setRules(newRules);
-            };
-
-            return (
-              <li key={i} className={styles.listitem}>
-                <div className="row align-items-center">
-                  <span className="ml-2 mb-2">
-                    {(i + 1) % 2 === 1 ? "Launch rule" : "Disable rule"}
-                  </span>
-                  <div className="col-sm-12 col-md mb-2 pl-2 pr-2">
-                    <SelectField
-                      name="date-operator"
-                      value={
-                        timestamp !== null
-                          ? "at a specific date and time"
-                          : enableFeature
-                          ? "immediately"
-                          : "manually"
-                      }
-                      options={(i + 1) % 2 === 1 ? launchOptions : endOptions}
-                      onChange={(value) => {
-                        if (value === "at a specific date and time") {
-                          onChange(new Date().toISOString(), "timestamp", i);
-                        } else {
-                          onChange(null, "timestamp", i);
-                        }
+          <li className={styles.listitem}>
+            <div className="row align-items-center">
+              <span className="ml-2 mb-2">Launch rule</span>
+              <div className="col-sm-12 col-md mb-2 pl-2 pr-2">
+                <SelectField
+                  name="date-operator"
+                  value={
+                    rules[0].timestamp === null
+                      ? "immediately"
+                      : "at a specific date and time"
```

Comment:
It's hard to tell by looking at this code if this will work or not.  You have to match a hard-coded string here against one on line 38.  I think it would be easier to read if you define these options inline.

Also, there's no reason to use this user-facing language as the option value.  You could instead do something like this:

```ts
[
  { label: "immediately", value: "" },
  { label: "at a specific date and time", value: "timestamp" },
]
```

That way you wouldn't have the hard-coded string `at a specific date and time` all throughout the code.

```ts
if (value === "at a specific date and time") {
```

becomes:

```ts
if (value) {
```

Reply:
Great feedback - this does ready a lot easier after applying these changes.